cfme/tests/containers/test_manageiq_ansible_users.py

```diff
@@ -0,0 +1,138 @@
+import pytest
+from cfme.containers.provider import ContainersProvider
+from cfme.configure.access_control import User
+from cfme.web_ui import Table
+from utils import testgen
+from utils.ansible import setup_ansible_script, run_ansible, \
+    fetch_miq_ansible_module, revert_basic_script
+from utils.appliance.implementations.ui import navigate_to
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.7")]
+pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
+
+table_el = '//table[contains(@class, ' \
```

Comment:
sure, I would advice to use users_table from cfme/configure/access_control.py untils this module will be fully converted to widgetastic.